src/pages/kwakjongbum/Main/KwakMain.js

```diff
@@ -1,7 +1,345 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import './KwakMain.scss';
+import CommentList from './CommentList';
 
 function KwakMain() {
-  return <div>곽종범 메인페이지</div>;
+  const [feedInfo, setFeedInfo] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/kwakjongbum/feedData.json')
+      .then(reponse => reponse.json())
+      .then(feedData => setFeedInfo(feedData));
+  }, []);
+
+  const user = {
+    user: 'wecode',
+  };
+  const [comment, setComment] = useState([]);
+  const [value, setValue] = useState('');
+  const [btn, setBtn] = useState(true);
+
+  const inputVal = e => {
+    setValue(e.target.value);
+  };
+
+  const btnActive = () => {
+    value.length > 0 ? setBtn(false) : setBtn(true);
+  };
+
+  const commentAdd = () => {
+    let copyComment = [...comment];
+    copyComment.push(value);
+    setComment(copyComment);
+    setValue('');
+  };
```

Comment:
```suggestion
  const commentAdd = () => {
    setComment([...comment, value]);
    setValue('');
  };
```
- spread를 활용해서도 할 수 있겠네요.